src/model/ServerNode.ts

```diff
@@ -46,7 +46,7 @@ export class ServerNode implements INode {
 
         const baseUrl = url.resolve(this.sastUrl, 'CxRestAPI/');
 
-        this.httpClient = new HttpClient(baseUrl, "Visual-Studio-Code", this.log);
+        this.httpClient = new HttpClient(baseUrl, "Visual-Studio-Code","", this.log);
```

Comment:
Below is constructor code     in httpClient.ts. I this proxy config param is after logger object so looks like order need to be corrected.


 constructor(private readonly baseUrl: string, private readonly origin: string, private readonly log: Logger, private proxyConfig?: ProxyConfig) {